lib/matplotlib/pyplot.py

```diff
@@ -2900,16 +2895,14 @@ def streamplot(
         x, y, u, v, density=1, linewidth=None, color=None, cmap=None,
         norm=None, arrowsize=1, arrowstyle='-|>', minlength=0.1,
         transform=None, zorder=None, start_points=None, maxlength=4.0,
-        integration_direction='both', broken_streamlines=True, *,
```

Comment:
This change seems strange.

Reply:
ooops,  Looks like I deleted it by accident.  Thanks for catching....

Reply:
Oh, hmmm.  I'm not doing this - its boilerplate.py.  I'm not actually clear what is going on here....

Reply:
Strange. Is boilerplate executed automatically or manually? (Sorry for my ignorance.)

Reply:
Manually.  This commit I've not executed boilerplate, and it seems better plus or minus the font-finding problem, which I believe is unrelated